### DISCUSSION TITLE
What's with the SPI Clock in 3.1.0?

### DISCUSSION THREAD

**Discussion by wayland** [shanteacontrols/OpenDeck#191] [Q&A] [answered]
I note that on the OpenDeck 3.1.0, RP2040 pin 9, GPIO 7, is marked SPI_CLK.  However, looking at the GPIO Functions table in the RP2040 datasheet, it's GPIO6 that's the SPI clock.  Is this an error?  

Datasheets used:
- https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf
- https://raw.githubusercontent.com/shanteacontrols/OpenDeck/master/bin/sch/opendeck/v3.1.0/OpenDeck-r3.1.0.pdf

Thanks!

**Comment by paradajz** [comment, marked as answer]
It is not an error. I am using bit banged SPI implementation so that I (or anyone else) can use any pins on a supported microcontroller for SPI parts such as the MAX led driver. Flexibility always comes with some downsides.
